app/models/nic/bootable.rb

```diff
@@ -8,28 +8,13 @@ class Bootable < Managed
 
     register_to_enc_transformation :type, lambda { |type| type.constantize.humanized_name }
 
-    def dhcp_record
-      return unless dhcp? or @dhcp_record
-      @dhcp_record ||= host.jumpstart? ? Net::DHCP::SparcRecord.new(dhcp_attrs) : Net::DHCP::Record.new(dhcp_attrs)
+    def initialize(*args)
+      logger.warn 'DEPRECATION WARNING: Nic::Bootable is replaced by Nic::Managed with provision: true, this class will be removed'
```

Comment:
does it make sense to use ActiveSupport::Deprecation.warn ? i saw we are inconsistent of our usage of deprecation messages.